tidb-cloud/data-service-response-and-status-code.md

```diff
@@ -17,7 +17,7 @@ Data Service returns an HTTP response with a JSON body. The response body contai
 - `data`: _object_. The execution results, which include three parts:
 
     - `columns`: _array_. Schema information for the returned fields.
-    - `rows`: _array_. The returned results in `key:value` format.
+    - `rows`: _array_. The returned results in `key:value` format. If **Batch** is enabled for an endpoint, the returned results of the endpoint also include the `"message"` and `"success"` fields for each row to indicate the response and status of that row.
```

Comment:
If **Batch operation** is enabled for an endpoint, the returned results of the endpoint will also include the `"message"` and `"success"` fields for each row to indicate their response and status.